### PR TITLE
Styled notes component

### PR DESCRIPTION
**Pull request by Tuasegun**

### CODE REVIEW

File: s/features/notes/components/notes/xiome-notes.css.ts [hunk 1]
```diff
@@ -1,18 +1,100 @@
-
-import {css} from "../../../../framework/component.js"
+import { css } from "../../../../framework/component.js";
 export default css`
+  :host {
```
Comment (chase-moskal) [inline s/features/notes/components/notes/xiome-notes.css.ts:3]: this file should use tabs, not spaces (it's currently spaces)

File: s/features/notes/components/notes/xiome-notes.ts [hunk 2]
```diff
@@ -1,156 +1,176 @@
-
-import styles from "./xiome-notes.css.js"
-import {makeNotesModel} from "../../models/notes-model.js"
-import {renderOp} from "../../../../framework/op-rendering/render-op.js"
-import {ComponentWithShare, mixinStyles, html} from "../../../../framework/component.js"
-import {ops} from "../../../../framework/ops.js"
+import styles from "./xiome-notes.css.js";
```
Comment (chase-moskal) [inline s/features/notes/components/notes/xiome-notes.ts:1]: the auto-formatting you used has changed this code so much, that we can't tell what substiantial changes have been made, making this very difficult to review

**Review by chase-moskal — CHANGES_REQUESTED** [review]
@Tuasegun thanks for contributing to the project.

can you please address the following issues?

- the history on this branch is quite convoluted.
    - we need to reconstitute the branch, so that it's one simple commit, based off the latest commit on `master`
    - one strategy to do this, from the `notesChanges` branch, would be to run `git reset master`, then creating a new commit from that point on master. it's a good practice to make a backup branch of your work just in case something goes wrong with the operation.
- the formatting of the code doesn't match the rest of the project.
    - this project uses tabs instead of spaces
    - this project doesn't use semicolons
    - the auto-formatting you've used has changed so many lines of code, it's very hard to distinguish which changes are actually substantial

once we these issues are fixed, we can proceed with code review.

File: s/features/notes/components/notes/xiome-notes.ts (hunk 2 above)
Comment (Tuasegun) [reply s/features/notes/components/notes/xiome-notes.ts:1]: I'll revert the formatting

**Pull request closed without merging**